### ISSUE TITLE
upgrade from 8.0.0 to 8.0.3

### ISSUE THREAD

**Issue by thiagotrennepohl** [sameersbn/docker-gitlab#575]
Hi.

Could someone help me? I'm stuck with this.

I updated gitlab from 6.7.3 to 8.0.0 with some trouble, but  searching around   i've  managed to update with success to 8.0.0

The thing is at version 8.0.3 everything looks fine, except my repos, gitlab says that the repo doesn't exist . pic below 
![no-repo](https://cloud.githubusercontent.com/assets/12124856/12425135/90e16400-beba-11e5-9abc-2a77457f6a85.png)

And the funny thing is when I create a new repo this happens
![no-repo2](https://cloud.githubusercontent.com/assets/12124856/12425158/a37f2bec-beba-11e5-9965-2cd46654ad06.png)

Tried with 8.0.2 as well  and it works fine
:(

I already ran ```app:sanitize```, ```db:migrate```, even ```gitlab:import:repos``` and my /home/git/data looks like this:
```
root@6db2785fb13a:/home/git/data# ls -lh
total 36K
drwxr-xr-x   2 git  git  4.0K Jan 19 12:57 backups
drwxr-xr-x   2 git  git  4.0K Jan 14 16:46 builds
drwxr-xr-x   3 git  git  4.0K Jan 14 15:34 config
drwxr-x--- 106 git  git  4.0K Jan 14 15:27 gitlab-satellites
drwxrws--- 156 git  git  4.0K Jan 19 10:32 repositories
drwxr-xr-x   4 git  git  4.0K Jan 14 16:46 shared
drwxr-xr-x   2 root root 4.0K Jan 14 16:46 ssh
drwxr-xr-x   4 git  git  4.0K Jan 19 15:40 tmp
drwxr-x---   6 git  git  4.0K Jan 14 15:27 uploads

```

``` gitlab:check ```
```
Repo base access is drwxrws---? ... no
  Try fixing it:
  sudo chmod -R ug+rwX,o-rwx /home/git/repositories/
  sudo chmod -R ug-s /home/git/repositories/
  find /home/git/repositories/ -type d -print0 | sudo xargs -0 chmod g+s
  For more information see:
  doc/install/installation.md in section "GitLab Shell"
  Please fix the error above and rerun the checks.
hooks directories in repos are links: ... 
Eduardo Nunes / site-template ... repository is empty
ci-platform / portal-site ... repository is empty
Paulo Ragonha / react-server-poc ... repository is empty
Eduardo Nunes / swig-server-poc ... repository is empty
Rodrigo Vieira / Dust POC ... repository is empty

```
my  ``` /home/git/repositories/ ``` is an empty folder.

Any clues?

Many thanks in advance.

PS: Same happens with 8.3.4

**Comment by sameersbn** [comment]
I am a little confused about your report. Do you mean with 8.0.0, 8.0.2 everything is okay? i.e. the repositories exist? and with 8.0.3 they are missing?

> my /home/git/repositories/ is an empty folder.

The repositories are actually located in `/home/git/data/repositories`. did not notice the `/home/git/repositories` directory was being created. You can ignore it as it is never used. Please inspect the contents of `/home/git/data/repositories` to check if the bare repositories exist.

Do you have a backup created with the `6.7.3` version (hope you do)? If you do, then please restore it on sameersbn/gitlab:6.7.3 and try upgrading in phases, i.e. upgrade to the last 6.x version, then upgrade to the last 7.x version, then to 8.0.5-1 version and finally to the `8.2.3` release. At every upgrade please check if the instance is working correctly and all your repos are there. It may also be worth taking backups along the way.

Let me know how it goes.

**Comment by thiagotrennepohl** [comment]
@sameersbn Thank you for the reply.

Correct, with 8.0.3 they are missing.

> I am a little confused about your report. Do you mean with 8.0.0, 8.0.2 everything is okay? i.e. the repositories exist? and with 8.0.3 they are missing?

> > my /home/git/repositories/ is an empty folder.

yes, the bare repositories exist.
> Please inspect the contents of /home/git/data/repositories to check if the bare repositories exist.

yes, i have, I'll do what you suggested :)
> Do you have a backup created with the 6.7.3 version (hope you do)?

 Anything i'll let you know, thank you :)

**Comment by thiagotrennepohl** [comment]
Hello.

Made it!

I tried to follow  your suggestion by upgrading to the last version of 6.x , last 7.x then 8.0.5-1, but i didn't succeed.

Well, i decided to run  a fresh container with an empty volume, just to create gitlab's 
(```git/data/ ```) folders in my host machine, then i  moved my repos folder from the old volume to the new volume (```/srv/gitlab/data/repositories ```).

I started a container with this new volume and ran ```app:sanitize``` to fix permissions,  and  then ```app:rake gitlab:import:repos```  and shazam, all users, commits and repos in the place they should be :)

PS: Still using the same database.
